src/comments/forms.py

```diff
@@ -1,6 +1,6 @@
 from django import forms
 
-from django_summernote.widgets import SummernoteWidget
+from utilities.widgets import SummernoteWidget
```

Comment:
If we are extending the SummernoteWidget, lets give it a new name to prevent confusion.  How about `BDSummernoteWidget` (ByteDeck Summernote Widget)

Reply:
This is confusing me.  I don't see a SummernoteWidget in utilities.widgets in this PR?  I think I'm getting confused by all the class renaming that's going on.